ProcessLib/HT/HTFEM.h

```diff
@@ -173,7 +218,8 @@ class HTFEM : public HTLocalAssemblerInterface
         auto const specific_heat_capacity_solid =
             _material_properties.specific_heat_capacity_solid(t, pos)[0];
 
-        auto const solid_density = _material_properties.density_solid(t, pos)[0];
+        auto const solid_density =
+            _material_properties.density_solid(t, pos)[0];
```

Comment:
Why not use
```
  solid_phase.property(MaterialPropertyLib::PropertyType::density)
                    .template value<double>(vars);
```

Reply:
You are right. Until now only the liquid phase properties and some solid phase properties are changed. I want to convert the remaining properties in a following PR. I'll update the PR description.

Reply:
I see.